### DISCUSSION TITLE
Precedence of redirects

### DISCUSSION THREAD

**Discussion by benoitblancquaert** [ethercreative/seo#353] [Q&A] [answered]
Do the redirects set via SEO take precedence over those in the .htaccess file located in the server’s web root?

**Comment by Tam** [comment, marked as answer]
No, the redirects are currently triggered during Craft's routing phase. This means redirects in your .htaccess file will take precedence as the server will read those before Craft is started.

**Reply by benoitblancquaert** [reply]
Thanks for clearing that up!
